_plugins/jekyll-multiple-languages-plugin.rb

```diff
@@ -0,0 +1,67 @@
+module Jekyll
+
+
+  ##############################################################################
+  # class Site
+  ##############################################################################
+  class Site
+
+    attr_accessor :translation_file_content   # Hash that stores parsed translations read from YAML files.
+    alias_method :process_extended, :process
+
+    def process
+      self.translation_file_content ||= {}
+      process_extended
+    end
+  end
+
+  module LocalizeTagExtensions
+    #======================================
+    # initialize
+    #======================================
+    def initialize(tag_name, key, tokens)
+      super
+      @key = key.strip
+    end
+    #======================================
+    # render
+    #======================================
+    def render(context)
+      if      "#{context[@key]}" != "" # Check for page variable
```

Comment:
Seems like an un-necessary interpolation here. Possible to use `context[@key].empty?` instead?